docs/current/quickstart/snippets/build/index.mjs

```diff
@@ -0,0 +1,32 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // use a node:16 container
+  // mount the source code directory on the host
+  // at /src in the container
+  const source = await client.container()
+    .from("node:16")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] } ))
```

Comment:
maybe refactor the `clien.host` call to a different variable so code is more readable?